### PR TITLE
Add fzf to arkade get

### PR DESCRIPTION
**Pull request by Shikachuu**
## Description
`fzf` is the most popular fuzzy find cli tool.
Also removed the duplicate kumactl (probably it was a merge leftover)

## Motivation and Context
- [ ] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?
Covered with unit tests.
`make e2e` passed.
Also tested with the following shell script:
```bash
#!/bin/bash

set -e -x -u -o pipefail

go build 

./arkade get fzf --arch arm64 --os darwin --quiet
file $HOME/.arkade/bin/fzf
rm $HOME/.arkade/bin/fzf

./arkade get fzf --arch x86_64 --os darwin --quiet
file $HOME/.arkade/bin/fzf
rm $HOME/.arkade/bin/fzf

./arkade get fzf --arch x86_64 --os linux --quiet
file $HOME/.arkade/bin/fzf
rm $HOME/.arkade/bin/fzf

./arkade get fzf --arch arm64 --os linux --quiet
file $HOME/.arkade/bin/fzf
rm $HOME/.arkade/bin/fzf

./arkade get fzf --arch armv7l --os linux --quiet
file $HOME/.arkade/bin/fzf
rm $HOME/.arkade/bin/fzf

./arkade get fzf --arch armv6l --os linux --quiet
file $HOME/.arkade/bin/fzf
rm $HOME/.arkade/bin/fzf

./arkade get fzf --arch x86_64 --os ming --quiet
file $HOME/.arkade/bin/fzf
rm $HOME/.arkade/bin/fzf

./arkade get fzf --arch arm64 --os ming --quiet
file $HOME/.arkade/bin/fzf
rm $HOME/.arkade/bin/fzf

./arkade get fzf --arch armv7l --os ming --quiet
file $HOME/.arkade/bin/fzf
rm $HOME/.arkade/bin/fzf

./arkade get fzf --arch armv6l --os ming --quiet
file $HOME/.arkade/bin/fzf
rm $HOME/.arkade/bin/fzf
```
<details>
<summary>Output</summary>
<pre><code>
/home/shika/.arkade/bin/fzf: Mach-O 64-bit arm64 executable, flags:<|DYLDLINK|PIE>
/home/shika/.arkade/bin/fzf: Mach-O 64-bit x86_64 executable
/home/shika/.arkade/bin/fzf: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=Ch5o9kNIIvTaJTliKgZU/OlxB46JDJUSZcH4EhslB/QxuQjCvGx9aQ84diXNo8/TueTdrn9ahEos8yXYrO5, stripped
/home/shika/.arkade/bin/fzf: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, Go BuildID=0bQ5QBFXILUT1YVkRNWG/AbBnYAaKlkGeVy0EuXHJ/LipobelVZtQeSsE_r43i/QmJ8Knj5qVV4h715FIXc, stripped
/home/shika/.arkade/bin/fzf: ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), statically linked, Go BuildID=agTZxLrb-xWx_Mfdw71R/L8RGbKA5tmdxzT3v_AjA/6YVr9g0ahV-LhLHh4HbX/rgVRb51x0O5piKg6nOTo, stripped
/home/shika/.arkade/bin/fzf: ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), statically linked, Go BuildID=wkg1cdYKr9v93uFUvtP1/yoX_fcnAz-EFoqTqnDIV/6YVr9g0ahV-LhLHh4HbX/bko2rZZHk65jKpzPP9ez, stripped
/home/shika/.arkade/bin/fzf: ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), statically linked, Go BuildID=wkg1cdYKr9v93uFUvtP1/yoX_fcnAz-EFoqTqnDIV/6YVr9g0ahV-LhLHh4HbX/bko2rZZHk65jKpzPP9ez, stripped
/home/shika/.arkade/bin/fzf: ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), statically linked, Go BuildID=wkg1cdYKr9v93uFUvtP1/yoX_fcnAz-EFoqTqnDIV/6YVr9g0ahV-LhLHh4HbX/bko2rZZHk65jKpzPP9ez, stripped
/home/shika/.arkade/bin/fzf: ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), statically linked, Go BuildID=wkg1cdYKr9v93uFUvtP1/yoX_fcnAz-EFoqTqnDIV/6YVr9g0ahV-LhLHh4HbX/bko2rZZHk65jKpzPP9ez, stripped
/home/shika/.arkade/bin/fzf: ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), statically linked, Go BuildID=wkg1cdYKr9v93uFUvtP1/yoX_fcnAz-EFoqTqnDIV/6YVr9g0ahV-LhLHh4HbX/bko2rZZHk65jKpzPP9ez, stripped
</code></pre>
</details>
## Are you a GitHub Sponsor (Yes/No?)

- [ ] Yes
- [X] No

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [X] I have updated the list of tools in README.md if (required) with `./arkade get -o markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [X] I have signed-off my commits with `git commit -s`
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

- [X] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

**Comment by alexellis** [comment]
Please remove armv6l. Why do you show a test for armv7l and Ming, which is Windows?

File: pkg/get/get_test.go [hunk 1]
```diff
@@ -3592,3 +3592,86 @@ func Test_DownloadHey(t *testing.T) {
...
+			os:      "ming",
+			arch:    arch64bit,
+			version: version,
+			url:     `https://github.com/junegunn/fzf/releases/download/0.30.0/fzf-0.30.0-windows_amd64.zip`,
+		},
+		{
+			os:      "ming",
+			arch:    "armv6l",
+			version: version,
+			url:     `https://github.com/junegunn/fzf/releases/download/0.30.0/fzf-0.30.0-windows_armv6.zip`,
```
Comment (alexellis) [inline pkg/get/get_test.go:3650]: Please remove

File: pkg/get/get_test.go [hunk 2]
```diff
@@ -3592,3 +3592,86 @@ func Test_DownloadHey(t *testing.T) {
...
+		},
+		{
+			os:      "ming",
+			arch:    archARM7,
+			version: version,
+			url:     `https://github.com/junegunn/fzf/releases/download/0.30.0/fzf-0.30.0-windows_armv7.zip`,
```
Comment (alexellis) [inline pkg/get/get_test.go:3656]: Please remove

File: pkg/get/get_test.go [hunk 3]
```diff
@@ -3592,3 +3592,86 @@ func Test_DownloadHey(t *testing.T) {
...
+		},
+		{
+			os:      "ming",
+			arch:    archARM64,
+			version: version,
+			url:     `https://github.com/junegunn/fzf/releases/download/0.30.0/fzf-0.30.0-windows_arm64.zip`,
```
Comment (alexellis) [inline pkg/get/get_test.go:3662]: We have no windows arm files so let's remove. Less is more

**Review by alexellis — CHANGES_REQUESTED** [review]
Let's reduce scope please. No arm for Windows and no armv6.

**Comment by Shikachuu** [comment]
> Please remove armv6l. Why do you show a test for armv7l and Ming, which is Windows?

[Because somehow they've got binaries for that.](https://github.com/junegunn/fzf/releases/tag/0.30.0)
But if you wish I can remove armv6l from every os and remove every windows version except arm64 and amd64.

**Comment by alexellis** [comment]
Thank you, please go ahead for this new CLI. We don't need to maintain things that people can't or won't use.

File: pkg/get/get_test.go [hunk 4]
```diff
@@ -3592,3 +3592,86 @@ func Test_DownloadHey(t *testing.T) {
...
+			os:      "linux",
+			arch:    archARM64,
+			version: version,
+			url:     `https://github.com/junegunn/fzf/releases/download/0.30.0/fzf-0.30.0-linux_arm64.tar.gz`,
+		},
+		{
+			os:      "linux",
+			arch:    "armv6l",
+			version: version,
+			url:     `https://github.com/junegunn/fzf/releases/download/0.30.0/fzf-0.30.0-linux_armv6.tar.gz`,
```
Comment (alexellis) [inline pkg/get/get_test.go:3632]: Let's just keep armv7 and arm64 please

Comment (Shikachuu) [reply pkg/get/get_test.go]: Done!

**Comment by Shikachuu** [comment]
> Please see comments

Done :heavy_check_mark:

**Comment by alexellis** [comment]
This PR needs to be rebased before we can merge it.

**Pull request closed without merging**